pubs/pubs_cmd.py

```diff
@@ -96,9 +99,12 @@ def execute(raw_args=sys.argv):
             parser.print_help(file=sys.stderr)
             sys.exit(2)
 
+        events.PreCommandEvent().send()
         args.prog = "pubs"  # FIXME?
         args.func(conf, args)
 
     except Exception as e:
         if not uis.get_ui().handle_exception(e):
             raise
+    finally:
+        events.PostCommandEvent().send()
```

Comment:
I am not sure whether we want to send it even when the command failed. Maybe we could send the success status too.

Anyway, this goes beyond this PR. Let stick with what is necessary here; we can always extend the logic later.

Reply:
I think we do, because on a command processing multiple entries, some may be modified until an error is encountered. And yes, we may want to signal that not everything is normal. But the git plugin should not care.